docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
+For some of these environments, such as CoreOS, you can actually install Flocker inside a container.
+
+.. note::
+   Installing Flocker in containers is useful for environments such as CoreOS, RancherOS, or Amazon ECS, but please note that doing so is currently experimental.
+
+You can find the relevant Docker Hub images here:
+
+* `Dataset Agent <https://hub.docker.com/r/clusterhq/flocker-dataset-agent/>`_
+* `Container Agent <https://hub.docker.com/r/clusterhq/flocker-container-agent/>`_
+* `Control Service <https://hub.docker.com/r/clusterhq/flocker-control-service/>`_
+* `Docker Plugin <https://hub.docker.com/r/clusterhq/flocker-docker-plugin/>`_
+
+Before you install Flocker in containers, all authentication must be completed.
+This means that you will have generated the cluster certificate and key, the  node certificates and keys, and the API certificates and keys.
+
+For more information on generating certificates, please see  :ref:`authentication`. 
+
+Before you begin, you will need to make sure port ``4523`` is available for the control API and port ``4524`` is available for agent nodes.
+
+Use the following steps to install Flocker using Docker containers:
+
+#. Run the following command on the hosts that are running your Flocker containers:
```

Comment:
:question: This step is only necessary when you start the containers with `-e DEBUG=TRUE` and I didn't find it very useful when I was working with Flocker coreos. Omit it?